src/web/components/Cards/Card/components/CardListItem.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { css } from 'emotion';
 
-const listStyles = css`
-    /* Here we ensure the card stretches to fill the containing space */
-    flex-basis: 100%;
-    /* display block here prevents that annoying bar at the bottom of the image problem
-    we might be able to remove this if this is already set in global css */
+const listStyles = (percentage?: CardPercentageType) => css`
     display: flex;
+    flex-grow: 1;
+
+    /* Here we ensure the card stretches to fill the containing space falling back to flex: 1 */
+    ${percentage ? `flex-basis: ${percentage && percentage};` : `flex: 1;`}
```

Comment:
Is `percentage && percentage` intentional?

Reply:
Ha, no. That's a copy paste error. This is fixed in a follow up PR.